state/txindex/kv/kv_test.go

```diff
@@ -101,6 +102,16 @@ func TestTxSearch(t *testing.T) {
 		{"account.number < 10000 AND account.owner = 'Ivan'", 1},
 		// search using a prefix of the stored value
 		{"account.owner = 'Iv'", 0},
+		// search for owner with dash in name
```

Comment:
Should this not be:

```suggestion
		// search for owner with slash in name
```

And similar for all the code comments that follow...

Reply:
You are right :D good catch!